core/ecschema-metadata/src/Metadata/EntityClass.ts

```diff
@@ -21,11 +21,12 @@ import { SchemaItem } from "./SchemaItem";
 
 /**
  * A Typescript class representation of an ECEntityClass.
- * @beta
+ * @public @preview
  */
 export class EntityClass extends ECClass implements HasMixins {
   public override readonly schemaItemType = EntityClass.schemaItemType;
   public static override get schemaItemType() { return SchemaItemType.EntityClass; }
+  /** @internal */
   protected _mixins?: LazyLoadedMixin[];
```

Comment:
We could make this property private. In can be accessed through the entityClass.mixins getter and is set through the addMixin setter method. I haven't any of our code using the field directly outside this class.

Reply:
This applies to a couple of more schema items. 
I don't want to repeat the same message again.